### DISCUSSION TITLE
Closed Issue #46: PackageConfig testing of info_url

### DISCUSSION THREAD

**Discussion by DavinChurch** [aplteam/Tatin#51] [Ideas]
You've closed issue #46 but I disagree with your comment. While true, that doesn't help the author get it right in the first place. The command could at least validate the URL at that moment to make sure that it was typed right, for instance. Just because it can change in the future doesn't mean that an immediate validation doesn't still have value.

**Comment by aplteam** [comment]
It's not a matter of changing. A URL might not be available right now, but 2 seconds later it might well be. Or not. And it might be available right now and never again. 

Other opinions?

**Comment by DavinChurch** [comment]
Perhaps an error is not necessary, but just a warning. I'm just thinking that authors like me could use some help avoiding typing mistakes.

**Comment by e9gille** [comment]
Sounds fair enough. A simple HEAD request and a warning if an error was returned.

**Comment by aplteam** [comment]
Implemented in 0.55.0

**Comment by aplteam** [comment]
Since there is no way to archive a discussion, or mark it as obsolete or whatever, I am going to lock this discussion.

That allows you specify `is:unlocked` in order to filter it out.
